features/integration/applications.feature

```diff
@@ -149,17 +149,17 @@ Feature: Applications
       And I fund the current application's address with 100000000 microalgos.
       Then I get the account address for the current application and see that it matches the app id's hash
 
-      # create a box called "str:name"
-      And I build an application transaction with the transient account, the current application, suggested params, operation "call", approval-program "", clear-program "", global-bytes 0, global-ints 0, local-bytes 0, local-ints 0, app-args "str:create,str:name", foreign-apps "", foreign-assets "", app-accounts "", extra-pages 0, boxes "0,str:name"
+      # create a box called "str:name" (i.e., b64:bmFtZQ==)
+      And I build an application transaction with the transient account, the current application, suggested params, operation "call", approval-program "", clear-program "", global-bytes 0, global-ints 0, local-bytes 0, local-ints 0, app-args "str:create,b64:bmFtZQ==", foreign-apps "", foreign-assets "", app-accounts "", extra-pages 0, boxes "0,b64:bmFtZQ=="
```

Comment:
I think it's totally ok to let the `app-args` be abi-encoded. I'm only asking for changes in the _parsed response_ assertions for box names and values.